### DISCUSSION TITLE
How to disable windows going partially transparent on focus change?

### DISCUSSION THREAD

**Discussion by PythonGermany** [Manjaro-Sway/manjaro-sway#726] [Q&A] [answered]
I recently installed manjaro-sway and I can't figure out how to disable windows switch to be transparent for a short amount of time each time I change focus to it. Is it possible to disable this?

**Comment by boredland** [comment, marked as answer]
The tool is called "flashfocus" and you can simply uninstall it!

**Reply by PythonGermany** [reply]
Ah, found it and disabled it. So it is an optional integration and can be disabled either by uninstalling it or as described here: [SUPPORT.md](https://github.com/Manjaro-Sway/manjaro-sway/blob/main/SUPPORT.md#disabling-integrations)
Thank you the fast solution and the great service you provide with manjaro-sway
